pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1108,6 +1108,27 @@ func (b *changefeedResumer) Resume(ctx context.Context, execCtx interface{}) err
 	return nil
 }
 
+// ensureClusterIDMatches verifies that this job record matches
+// the cluster ID of this cluster.
```

Comment:
could help to mention the context (streaming replication) in which this would error

Reply:
extended comment.